libs/transactions/src/lib/views/transaction-details-view/transaction-details-view.component.scss

```diff
@@ -0,0 +1,4 @@
+.header {
```

Comment:
I would recommend using BB design system classes or in this case Bootstrap classes instead of creating the custom style.